audit-base/pom.xml

```
@@ -32,6 +32,11 @@
             <artifactId>lombok</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
+            <version>${jakarta.annotation-api.version}</version>
```

We may omit the version, defined in the Spring Boot BoM.

```suggestion
            <artifactId>jakarta.annotation-api</artifactId>
```

---

Good idea.